go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/streadway/amqp v0.0.0-20180315184602-8e4aba63da9f
 	github.com/stretchr/testify v1.7.0
 	github.com/uber/jaeger-client-go v2.22.1+incompatible
-	github.com/uber/jaeger-lib v2.2.0+incompatible
+	github.com/uber/jaeger-lib v2.4.2-0.20210604143007-135cf5605a6d+incompatible
```

Comment:
When do you expect this change to be released officially?

Reply:
I was waiting for a new release to open this PR, but the fix has been merged since April. I'll keep an eye on it.

Reply:
@peterklijn this PR? If yes as soon as we can have approvals from the rest of the team. @drakos74 @tpaschalis

Reply:
I think that's referring to a new jaeger-lib release, since this is pinned on commit `135cf56` on the jaeger-lib repo, right?

I don't think there's any danger of us breaking things here with some `go get` command in the future, so  I believe we can proceed with merging here.

Reply:
Sorry, I meant when do you expect jaeger-lib 2.4.2 to be released. I noticed your PR has been merged to master, but the version you use in go.mod does not point to an official release.

Reply:
@peterklijn  you have a point. Either we go to an official 2.4.1 is the latest release or leave the existing one as it is. We should not use an unofficial release.

Reply:
Since we measure request latencies on a `Timer` metric type, as seen [here](https://github.com/jaegertracing/jaeger-client-go/blob/master/rpcmetrics/metrics.go#L38), going with the 2.4.1 version is not enough to close this Patron issue. Should we wait for a new release?

Reply:
Is this unrelated with the existing PR? The current is only about making the buckets configurable. If we need to go to the newest i would wait for the next official release if you don't mind.

Reply:
This PR aimed to make the buckets configurable for all metric types that use them. i.e. `Histogram`, `Timer`.
Using version 2.4.1, the fix will be applied only to the first type, which does not close the linked Patron issue. 
Let's wait for an official release then - no problem with that.

Reply:
We've talked this through, and we decided that it's okay to go with the pinned commit for now. 

When a new version of `jaeger-lib` hits the ground, we can just go mod tidy at a later time!